websites/W/Wolfery/metadata.json

```
@@ -0,0 +1,32 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "166001598802296833",
+		"name": "drraccoony"
+	},
+	"service": "Wolfery",
+	"description": {
+		"en": "Support for online roleplaying tool, Wolfery "
```

Description should be a description of the website, not the presence, it seems like their official description is "A textual world of roleplay"
```suggestion
		"en": "A textual world of roleplay "
```